### PR TITLE
Fix test_events.py

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: cfme/tests/cloud_infra_common/test_events.py -v --long-running --use-provider complete --event-testing}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 04976584f8178996e3ae2982f3b8497eb6b1ffc2:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
Only some provider errors + it obviously does not work on 5.5 ...

File: cfme/tests/cloud_infra_common/test_events.py [hunk 1]
```diff
@@ -21,20 +20,15 @@ def pytest_generate_tests(metafunc):
     testgen.parametrize(metafunc, argnames, argvalues, ids=idlist, scope="module")
 
 
-@pytest.fixture(scope="module")
+@pytest.yield_fixture(scope="function")
```
Comment (RonnyPfannschmidt) [inline cfme/tests/cloud_infra_common/test_events.py]: scope change?

Comment (mfalesni) [inline cfme/tests/cloud_infra_common/test_events.py]: Yes, because I want the VM to be provisioned per test.

**Pull request merged**